app/assets/stylesheets/pages/_home.scss

```diff
@@ -1 +1,48 @@
-// Specific CSS for your home-page
+body {
+  margin: 0 !important;
+}
+
+.container {
```

Comment:
I wouldn't label this container as bootstrap has a inbuilt class of container already